pbin.sh

```diff
@@ -100,13 +106,13 @@ set_defaults() {
 	unset reply
 
 	# default to user@hostname
-	name=${SUDO_USER:-$USER}@${HOSTNAME:-$(hostname)}
+	name=${SUDO_USER:-$USER}
```

Comment:
But people won't know to do that and what does it matter if we have the full thing?